adventure/adventure.py

```diff
@@ -3273,18 +3273,18 @@ def check_files():
 
     files = {'teams.json' : {}, 'settings.json' : {}}
     for file, default in files.items():
-        if not os.path.isfile(base_path + file):
```

Comment:
there are some functionalilty choices you're sprinking in this PR as well.
This makes it slower for me to check this large PR.
If you had made smaller PRs that encompassed each file, I'd be able to approve them iteratively and have those changes in master quicker.